include/deal.II/numerics/data_postprocessor.h

```diff
@@ -188,17 +190,37 @@ namespace DataPostprocessorInputs
      * called by DataOut and similar classes when creating the object that
      * is then passed to DataPostprocessor.
      */
-    template <typename DoFHandlerType>
+    template <int dim>
     void
+    set_cell(const typename DoFHandler<dim, spacedim>::cell_iterator &cell);
+
+    /**
+     * @copydoc CommonInputs::set_cell<dim>()
```

Comment:
doxygen doesn't like template parameters in copydoc targets. Do you know how to make this work? Otherwise I'll simply copy the doc.

Reply:
> Otherwise I'll simply copy the doc.

Sounds good!